desktop-app/app/actions/browser.js

```diff
@@ -322,6 +324,37 @@ export function triggerScrollDown() {
 
 export function screenshotAllDevices() {
   return (dispatch: Dispatch, getState: RootStateType) => {
+    const {
+      browser: {
+        zoomLevel,
+        previewer,
+        filters
+      }
+    } = getState();
+
+    // force reset filters
+    dispatch(
+      newFilters({
```

Comment:
@sprabowo I think these filters, layouts, and zoom level changes should to reset to the value before the screen was triggered. So that the user can continue using it from where they left it. 

What do you think?

Reply:
@manojVivek agree with this. but i'm still confusing to reset and the lifecycle. resetting can't be done from the same actions. because dispatch can't wait for pubsub.publish.

Reply:
@sprabowo Yeah agreed, the current pubsub implementation doesn't return the result from the subscribers back. We will have to find a way to do this. 

Meanwhile, can you please attach a screencast of how this works to the PR? So that we can merge the current improvements.

Reply:
hi @manojVivek i just done reswetting the default layout. please review it.

this will fix: 
1. messed when screenshoot
2. create custom preview layout, zoomlevel then resetting to previous user preference

[Gojek - Kindle Fire HDX - 2020-10-6 at 11 18 53 AM](https://user-images.githubusercontent.com/11748183/84226856-1a286180-ab0d-11ea-8eda-5104d4285cb2.png)
[Gojek - iPhone X - 2020-10-6 at 11 18 53 AM](https://user-images.githubusercontent.com/11748183/84226845-13015380-ab0d-11ea-98b2-2bcb705a16fe.png)
[Gojek - Laptop - 2020-10-6 at 11 18 53 AM](https://user-images.githubusercontent.com/11748183/84226873-27455080-ab0d-11ea-9bb3-e2563c182c35.png)
[Gojek - iPhone 6-7-8 Plus - 2020-10-6 at 11 18 53 AM](https://user-images.githubusercontent.com/11748183/84226837-0b41af00-ab0d-11ea-9376-58ec76fbe954.png)

[Freedomhot - iPad - 2020-10-6 at 11 23 48 AM](https://user-images.githubusercontent.com/11748183/84226794-f5cc8500-ab0c-11ea-9ffb-068fd82c0dd3.png)
[Freedomhot - iPhone 6-7-8 Plus - 2020-10-6 at 11 23 48 AM](https://user-images.githubusercontent.com/11748183/84226803-f9f8a280-ab0c-11ea-846f-617a63011995.png)
[Freedomhot - iPhone X - 2020-10-6 at 11 23 48 AM](https://user-images.githubusercontent.com/11748183/84226806-fb29cf80-ab0c-11ea-8193-04addf88f888.png)
[Freedomhot - Kindle Fire HDX - 2020-10-6 at 11 23 48 AM](https://user-images.githubusercontent.com/11748183/84226809-fc5afc80-ab0c-11ea-8afb-737c59de376e.png)
[Freedomhot - Laptop - 2020-10-6 at 11 23 48 AM](https://user-images.githubusercontent.com/11748183/84226814-fe24c000-ab0c-11ea-9e5e-f9e2ac099b9c.png)

dont forget to watch this screencast https://www.youtube.com/watch?v=zN2vanuwd1k

Reply:
There's a bunch of whitespace on the bottom of all the screenshots. The freedomhot ones have a LOT.

Reply:
@avimar
It is still better solution than messed up. If there's no whitespace. The rest of scroll viewport will screenshoot twice (see issue opened by author). For whitespace i think you can crop later

Maybe you can improve this PR if you had better solution to remove whitespace